app/views/splash/index.html.erb

```diff
@@ -5,7 +5,11 @@
     <h1><%= application_name %></h1>
     <p><%= t('application.tagline') %></p>
     <p>
-      <%= link_to t('hyku.splash.account_signup'), account_sign_up_path, class: 'btn btn-lg btn-sign-up' %>
+      <% if !Settings.multitenancy.admin_only_tenant_creation || can?(:manage, Account) %>
+        <%= link_to t('hyku.splash.account_signup'), account_sign_up_path, class: 'btn btn-lg btn-sign-up' %>
+      <% elsif !user_signed_in? %>
```

Comment:
And if the above condition is false AND I am signed in... I see nothing here, right?

Reply:
Correct. Admittedly, I didn't think to much about that scenario. We could instead display a message along the lines of "You do not have permissions to create tenants".

Reply:
I've now updated this logic to display a "You are not authorized to create tenants" message in this specific scenario (and added specs to prove it out).